### PR TITLE
NOTICK Merge OS 4.6 to 4.7 2020-09-04

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

**Review by adelel1 — APPROVED** [review]
My updates look OK.

**Review by nikinagy — APPROVED** [review]
verify with NETWORK_PARAMETERS role and test changes look good to me

File: node/src/test/kotlin/net/corda/node/services/vault/VaultQueryTests.kt [hunk 1]
```diff
@@ -2776,6 +2782,8 @@ abstract class VaultQueryTestsBase : VaultQueryParties {
         }
     }
 
+    //linus one OOM issue
```
Comment (blsemo) [inline node/src/test/kotlin/net/corda/node/services/vault/VaultQueryTests.kt:2785]: Typo - linux, not linus
Also worrying...

**Review by blsemo — APPROVED** [review]
Ok

**Pull request merged**